tahoe_lti/xblock_helpers.py

```diff
@@ -0,0 +1,24 @@
+"""
+Helpers to access the XBlock Runtime modules.
+"""
+
+
+def get_xblock_user(xblock):
+    """
+    Gets the current request user for an XBlock instance.
+
+    :param xblock: XBlock instance.
+    :return: User or None.
+    """
+    from django.contrib.auth import get_user_model
```

Comment:
This import doesn't really need to be inside the function. It should be fine at top-level.

Reply:
I'm putting all imports inside the functions in this repo to avoid any import issues. This repo runs pretty deep in the system and hard to debug, so I'm using a very defensive programming style.